FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1571,4 +1571,81 @@ public function testPseudoSalesAreExportedCorrectly($pseudoSales, $expectedSales
             $values->getValues()
         );
     }
+
+    public function testCoverImageIsExportedAsFirstImage()
+    {
+        $articleConfiguration = [
+            'name' => 'FindologicArticle 1',
+            'active' => true,
+            'tax' => 19,
+            'supplier' => 'Findologic',
+            'categories' => [
+                ['id' => 3],
+                ['id' => 5]
+            ],
+            'images' => [
+                [
+                    "mediaId" => 2, // Muensterlaender_Lagerkorn_Ballons_Hochformat
+                    'main' => 0,
+                    'position' => 1,
+                ],
+                [
+                    "mediaId" => 3, // Muensterlaender_Lagerkorn_Imagefoto
```

Comment:
Maybe add somewhere in the test that these ids are related to the Shopware demo data, so it is clear to someone who is new to this.